tsdb/chunkenc/chunk.go

```diff
@@ -29,6 +30,8 @@ func (e Encoding) String() string {
 		return "none"
 	case EncXOR:
 		return "XOR"
+	case EncHisto:
+		return "Histo"
```

Comment:
Vere general and not very relevant thought: Is "Histo" a good abbreviation? It feels to long for an abbreviation. I would either go with the full name "Histogram" or just "His" if you want it short.

Reply:
or maybe "SHS" (for sparsehistogram) ?